pkg/capi/infrastructure/aws/aws.yaml

```diff
@@ -0,0 +1,186 @@
+## Cluster configs
+apiVersion: cluster.x-k8s.io/v1alpha3
```

Comment:
We'll either want to roll these changes back out to the `cluster-api-templates` repo or get rid of that repo and use this one as a source of truth. If we do the latter, we'll need to get the multi-az and autoscaling templates worked in here too. Just something to note for the future.